dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/impl/MediaTypeHeaderProvider.java

```diff
@@ -205,34 +204,34 @@ private static MediaType handleMediaTypeWithoutSubtype(String mType) {
         throw new IllegalArgumentException("Media type separator is missing");
     }
 
-    //Liberty change start
-    private static boolean isValid(String str)
-    {
+    // Determines whether the type or subtype contains any of the tspecials characters defined at:
```

Comment:
It looks like all of the `Liberty change` comments are removed from this class - so, in theory, it should match the version in CXF 3.4.3.  Can we just delete this overlay file?

Reply:
Talked with Andy.  There are a few other differences.  I am writing a PR to get those included in CXF so this part can be removed from Liberty on the next upgrade.

Reply:
Upon further review, there is no longer a need to keep this class in the Liberty fork.  Removing.